### DISCUSSION TITLE
Install Netbox without SSL

### DISCUSSION THREAD

**Discussion by arlenjenks** [netbox-community/netbox#7029] [Q&A] [answered]
I don't need SSL on my netbox install so I skipped that step in my install, now when I try to restart nginx I get the following error.

● nginx.service - A high performance web server and a reverse proxy server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Tue 2021-08-24 18:57:31 UTC; 53s ago
       Docs: man:nginx(8)
    Process: 31747 ExecStartPre=/usr/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=1/FAILURE)

Aug 24 18:57:31 ttcscitss15 systemd[1]: Starting A high performance web server and a reverse proxy server...
Aug 24 18:57:31 ttcscitss15 nginx[31747]: nginx: [emerg] cannot load certificate "/etc/ssl/certs/netbox.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No suc>
Aug 24 18:57:31 ttcscitss15 nginx[31747]: nginx: configuration file /etc/nginx/nginx.conf test failed
Aug 24 18:57:31 ttcscitss15 systemd[1]: nginx.service: Control process exited, code=exited, status=1/FAILURE
Aug 24 18:57:31 ttcscitss15 systemd[1]: nginx.service: Failed with result 'exit-code'.
Aug 24 18:57:31 ttcscitss15 systemd[1]: Failed to start A high performance web server and a reverse proxy server.

**Comment by jmcguir** [comment, marked as answer]
Hi,

Your ngnix config file is not correct. I'd suggest you check that and go
from there. I suspect you have a line about a cert.

See the error message that says your configuration wasn't valid.

Thanks,

Joel McGuire
